shell/platform/darwin/ios/ios_surface.h

```diff
@@ -36,8 +36,6 @@ class IOSSurface : public ExternalViewEmbedder {
 
   ExternalViewEmbedder* GetExternalViewEmbedderIfEnabled();
 
-  bool ResourceContextMakeCurrent();
```

Comment:
LGTM.

The implementation has now drifted a bit from the Android side. I don't think that's necessarily a problem, but it does make things a bit harder to understand on both sides.  Not sure if it's worth refactoring Android the way iOS was refactored.

Reply:
Probably. It may be worth having an embedder support library for this.